### ISSUE TITLE
timeline: items editable configuration

### ISSUE THREAD

**Issue by JGdijk** [seveves/ng2-vis#30]
I have a setup where i want certaint items not to be moved over to other groups, while others can. So in my visTImelineOptions i have set editable: true and on the specific items that should not be moved between groups i set the item on 
```
item.editable = {
                        remove: true,
                        updateGroup: false,
                        updateTime: true
                    }
```

this doesn't work, the item is still able to move between groups. I can set the item.editable to false completely, and it will not be editable anymore, but i only want it to be restricted in moving between groups. 

also when i look into the DataItem interface the editable value is only set to be a boolean, not a boolean **or**  an object.

**Comment by seveves** [comment]
I'm not sure if this is an issue with ng2-vis but with vis itself. May you clarify this? Maybe the feature you need is a new one of vis version 4.19?
If there is something missing in the DataItem inteface (or in any other Typescript definition file) then please provide a PR over at the [types repository for visjs](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/vis).

**Comment by JGdijk** [comment]
Ill see if I can reproduce this in visjs this weekend. Ill let you know

**Comment by JGdijk** [comment]
It is indeed implemented in visjs 4.18 and higher and ng2-vis is working with 4.17 currently, i will close this issue. Thanks for your work on ng2-vis.
